### ISSUE TITLE
Process for making global decisions

### ISSUE THREAD

**Issue by diatomsRcool** [TranslatorIIPrototypes/SRIGovernance#28]

**Comment by diatomsRcool** [comment]
Is this also a function of TACT? @cbizon ?

**Comment by cbizon** [comment]
No, I don't think so.  The issue is that TACT doesn't necessarily have input from every team so we don't want to say that it can make these decisions.

**Comment by diatomsRcool** [comment]
If we had to make a global decision, what would be the process? Would we use TACT?
